internal/core/arg_file_content.go

```diff
@@ -0,0 +1,56 @@
+package core
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"reflect"
+	"strings"
+
+	"github.com/scaleway/scaleway-sdk-go/strcase"
+)
+
+// loadArgsFileContent will hydrate args with default values.
+func loadArgsFileContent(cmd *Command, cmdArgs interface{}) error {
+	for _, argSpec := range cmd.ArgSpecs {
+		if !argSpec.CanLoadFile {
+			continue
+		}
+
+		fieldName := strcase.ToPublicGoName(argSpec.Name)
+		fieldValues, err := getValuesForFieldByName(reflect.ValueOf(cmdArgs), strings.Split(fieldName, "."))
+		if err != nil {
+			continue
```

Comment:
Maybe we should return an error and not fail silently ?

Reply:
validate.go line 48 et 78 also execute getValuesForFieldByName, log an info and continue.
It would not be too much to log the same info many times ?